spec/features/user_becomes_backer_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "spec_helper"
+
+feature "Subscribing to donate monthly" do
+  let(:stripe_helper) { StripeMock.create_test_helper }
+  before do
+    StripeMock.start
+    stripe_helper.create_plan(id: "planningalerts-backers-test-1", amount: 1)
+  end
+
+  after { StripeMock.stop }
+
+  given(:email) { "mary@local.org" }
+
+  it "isn't possible without javascript", pending: true do
+    visit new_subscription_path(email: email)
+
+    expect(page).to have_button("Donate each month", disabled: true)
+    expect(page).to have_content "Our payment form requires javascript from stripe.com"
+  end
+
+  context "with javascript" do
+    before do
+      # This is a hack to get around the ssl_required? method in
+      # the application controller which redirects poltergeist to https.
+      allow(Rails.env).to receive(:development?).and_return true
```

Comment:
This wise idea stuffs the whole test run, leading to the db locking up. I need another way to get around this.